OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -897,27 +900,34 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Millard2012AccelerationMuscle, Muscle);
                                           const AccelerationMuscleInfo& ami,
                                           std::string& caller) const;
 
-    //=====================================================================
-    // Private Utility Class Members
-    //      -Computes activation dynamics and fiber kinematics
-    //=====================================================================
+    // Status flag returned by initMuscleState().
+    enum ResultOfInitMuscleState {
+        Success_Converged,
+        Warning_FiberAtLowerBound,
+        Failure_MaxIterationsReached
+    };
+
+    /* Calculate the muscle state such that the fiber and tendon are developing
+    the same force.
 
-    /*
-    @param s the system state
+    @param result reference to the result vector: [solution error (N),
+           iterations, fiber length (m), fiber velocity (m/s),
+           passive force (N), tendon force (N)]
+    @param s the system state (const)
     @param aActivation the initial activation of the muscle
     @param aSolTolerance the desired relative tolerance of the equilibrium 
            solution
-    @param aMaxIterations the maximum number of Newton steps allowed before
-           attempts to initialize the model are given up, and an exception is 
-           thrown.
+    @param aMaxIterations the maximum number of Newton steps allowed before we
+           give up attempting to initialize the model
     @param aNewtonStepFraction the fraction of a Newton step to take at each
            update
     */
-    SimTK::Vector initMuscleState(SimTK::State& s, double aActivation,
-                             double aSolTolerance, int aMaxIterations,
-                             double aNewtonStepFraction) const;
-    
-
+    ResultOfInitMuscleState initMuscleState(SimTK::Vector& result,
+                                            const SimTK::State& s,
+                                            double aActivation,
+                                            double aSolTolerance,
+                                            int aMaxIterations,
+                                            double aNewtonStepFraction) const;
 
```

Comment:
> Could make the parameters `const` if they represent constant values inside the function. Compiler will help if function violates const-ness.

I generally do this in my own code but the author of these models did not follow this convention. I have made these parameters `const` but did not apply to the other methods in these classes; if more time is invested in this code, I think algorithmic improvements would provide greater impact.